### DISCUSSION TITLE
Override global css | Nuxt Theme / Extends | Tailwindcss Module

### DISCUSSION THREAD

**Discussion by ciarantr** [nuxt/nuxt#16342] [Questions]
Hi all, it possible to have different global css files load based on routes using the extends option?

I am trying to create a multi app and would need to remove/add different global css files based on the route, with the tailwind  module I need to have a global css file in the base nuxt config file.

## File structure ex.

<img width="771" alt="nuxt-mini-apps" src="https://user-images.githubusercontent.com/58338331/189520967-b5e59410-9923-4044-a19d-756e906278ab.png">

## nuxt config ex.

```js

import { defineNuxtConfig } from 'nuxt'

export default defineNuxtConfig({
  
  src: '.src/',
  extends: ['./app-1'],

  components: false,

  modules: ['@nuxtjs/tailwindcss'],
  tailwindcss: {
    cssPath: '~/assets/css/base.css',
  },
```

**Comment by sevillaarvin** [comment]
If you need a different css file based on the route/page, how about `useHead`?

https://v3.nuxtjs.org/getting-started/seo-meta#usehead-composable

**Comment by iamsurelydaveydave** [comment]
How to extends nuxt app using a private github repository? I tried it but I got an 404 Not Found message on the terminal.

**Comment by Zielgestalt** [comment]
I think you have to resolve the link in the base app. Relative Links only work from the child app ‚point of view‘. 

Example in the docus theme: https://github.com/nuxt-themes/docus/blob/dev/nuxt.config.ts

@, ~,… (in the base app) is not supported with extend at the moment.
